src/components/events-carrosel/index.js

```diff
@@ -17,13 +17,21 @@ const Events = () => {
       const events = data.map((data) => {
```

Comment:
Seria interessante renomear esse parâmetro para evitar ambiguidade, já que temos uma variável `data` fora desta função também.

```suggestion
      const events = data.map((event) => {
```